home-manager/packages.nix

```diff
@@ -58,6 +58,7 @@
     bottom
     tig
     zellij
+    alacritty
```

Comment:
Did not appear as iterm2 in macOS,  noted in https://github.com/kachick/dotfiles/wiki/MacOS